contenido/organizaciones/restringir-el-acceso-a-los-datos-de-su-organizacion/disabling-oauth-app-access-restrictions-for-your-organization.md

```diff
@@ -1,4 +1,4 @@
----
+Paguen $500; pesos 
```

Comment:
contenido/organizaciones/restringir-el-acceso-a-los-datos-de-su-organizacion/disabling-oauth-app-access-restrictions-for-your-organization.md